projects/projectsRoutes.js

```diff
@@ -0,0 +1,109 @@
+const express = require("express");
+
+const projects = require("./projectsModel.js");
+const actions = require("../actions/actionsModel.js");
+
+const router = express.Router();
+
+//=============== PROJECT ENDPOINTS =============== //
+
+// get a list of projects
+router.get("/", (req, res) => {
+  projects
+    .find()
+    .then(projects => {
+      res.status(200).json(projects);
+    })
+    .catch(err => res.status(500).json(err));
+});
+
+//get a project by its ID
+router.get("/:id", async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const project = await projects.findById(id);
+
+    if (project) {
+      res.status(200).json(project);
+    } else {
+      res.status(404).json({ message: "Project not found" });
+    }
+  } catch (error) {
+    res.status(500).json(error);
+  }
+});
+
+//Get actions of a specific project
+router.get("/:id/actions", (req, res) => {
+  actions
+    .findActionByProject(req.params.id)
```

Comment:
I'm getting an error; `actions.findActionByProject is not a function`. We can look at this during 1:1.